### ISSUE TITLE
Do you have the plan to release the pretraining code?

### ISSUE THREAD

**Issue by CVStack** [ucasligang/SemMAE#3]
Hi.

Thank you for sharing the paper and code. 

I want to refer the code for adaptive masking part.  

So, i am wondering if you have the plan to release your pretraining code.

**Comment by ucasligang** [comment]
Thank you for your attention to our work, please give me one or two more days.

**Comment by ucasligang** [comment]
> Hi.
> 
> Thank you for sharing the paper and code.
> 
> I want to refer the code for adaptive masking part.
> 
> So, i am wondering if you have the plan to release your pretraining code.

Hi, I have released our pertaining code. But there still be some work to do, such as uploading the attention mask, uploading the training script, double-checking the code and so on. I'll try to update the rest in a day.

**Comment by CVStack** [comment]
Thank you!
